packages/@vue/cli-plugin-e2e-webdriverio/README.md

```diff
@@ -0,0 +1,79 @@
+# @vue/cli-plugin-e2e-webdriverio
+
+> e2e-webdriverio plugin for vue-cli
+
+## Injected Commands
+
+- **`vue-cli-service test:e2e`**
+
+  Run end-to-end tests with [WebdriverIO](https://webdriver.io/).
+
+  Options:
+
+  ```
+  --remote          Run tests remotely on SauceLabs
+
+  All WebdriverIO CLI options are also supported.
+
+  ```
+
+  Additionally, all [WebdriverIO CLI options](https://webdriver.io/docs/clioptions.html) are also supported.
+  E.g.: `--baseUrl`, `--bail` etc.
+
+
+## Project Structure
+
+The following structure will be generated when installing this plugin:
+
+```
+tests/e2e/
+  ├── pageobjects/
+  |   └── app.js
+  ├── specs/
+  |   ├── app.spec.js
+  └── globals.js
+```
+
+In addition to that there will be 3 configuration files generated:
+
+- `wdio.shared.conf.js`: a shared configuration with all options defined for all environments
+- `wdio.local.conf.js`: a local configuration for local testing
+- `wdio.sauce.conf.js`: a remote configuration for testing on a cloud provider like [Sauce Labs](https://saucelabs.com/)
+
+The directories contain:
+
+#### `pageobjects`
+Contains an example for an page object. Read more on using [PageObjects](https://webdriver.io/docs/pageobjects.html) with WebdriverIO.
+
+#### `specs`
+Your e2e tests.
+
+## Installing in an Already Created Project
+
+``` sh
+vue add e2e-nightwatch
```

Comment:
```suggestion
vue add e2e-webdriverio
```

Reply:
You also need to edit https://github.com/vuejs/vue-cli/blob/dev/packages/@vue/cli-shared-utils/lib/pluginResolution.js#L5 to add this plugin to the official plugin list.

Better add a note here that for users of older CLI versions, they may need to run `vue add @vue/e2e-webdriverio`, because the official plugin list was hard-coded.